code/CloudFlare_Notifications.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class CloudFlare_Notifications extends Object
+{
+
+    /**
+     * Sets the X-Status header which creates the toast-like popout notification
+     *
+     * @param $message
+     */
+    protected static function setToast($message)
+    {
+        Controller::curr()->response->addHeader('X-Status', rawurlencode('CloudFlare: ' . $message));
+    }
+
+    /**
+     * Sets an Alert that will display on the CloudFlare LeftAndMain
+     *
+     * @param        $message
+     * @param string $type
+     */
+    protected static function setAlert($message, $type = 'success')
+    {
+        $jar = CloudFlare::singleton()->getSessionJar();
+
+        $jar['CFMessage'] = $message;
+        $jar['CFType'] = $type;
+
+        CloudFlare::singleton()->setSessionJar($jar);
```

Comment:
Seems a bit odd to me that you kind of reference these classes in a circular manner